install.sh

```diff
@@ -104,17 +67,49 @@ else
   echo_info "Unable to detect system information, installation will continue."
 fi
 
+# Do not redundantly use sudo if the user already owns the directory.
+if [ "$(stat -c '%u' "${CREW_PREFIX}")" == "$(id -u)" ]; then
+  # This will allow things to work without sudo.
+  sudo chown "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
+fi
+
+# Check if the user owns the CREW_PREFIX directory, as sudo is unnecessary if this is the case.
+# Check if the user is on ChromeOS v117+ and not in the VT-2 console, as sudo will not work.
+if [[ "$(stat -c '%u' "${CREW_PREFIX}")" == "$(id -u)" ]] && sudo 2>&1 | grep -q 'no new privileges'; then
+  echo_error "Please run the installer in the VT-2 shell."
+  exit 1
+fi
+
+# Check if the script is being run as root.
+if [ "${EUID}" == "0" ]; then
+  echo_error "Chromebrew should not be installed or run as root."
+  echo_info "Please login as 'chronos' and restart the install."
```

Comment:
`Please login to VT-2 as 'chronos' and restart the install.`?  Might be good to provide instructions too.  `To start the VT-2 session, type Ctrl + Alt + ->`.